contrib/kserve/models-web-app/base/kustomization.yaml

```diff
@@ -13,7 +13,7 @@ commonLabels:
 images:
 - name: kserve/models-web-app
   newName: kserve/models-web-app
-  newTag: v0.7.0
+  newTag: v0.8.0
```

Comment:
There is a mismatch between the tag here (0.8.0) and the one listed in the readme (0.8.1)

Reply:
Indeed, but there wasn't any change in the image of the models web app between `v0.8.0` and `v0.8.1` so I think it's OK